doc/classes/FoldableContainer.xml

```diff
@@ -287,6 +287,15 @@
 		</constant>
 	</constants>
 	<theme_items>
+		<theme_item name="arrow_collapsed_color" data_type="color" type="Color" default="Color(0.226, 0.478, 0.921, 1)">
+			The arrow's icon color when collapsed.
+		</theme_item>
+		<theme_item name="arrow_hover_color" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 1)">
+			The arrow's icon color when hovered and expanded.
+		</theme_item>
+		<theme_item name="arrow_normal_color" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 0.8)">
+			The arrow's icon color expanded.
```

Comment:
(cough) used when expanded, I will fix it later when i cherry-pick some changes from godot's foldable version ;P